app/views/authentication_mailer/sign_in_email.text.erb

```diff
@@ -1,5 +1,5 @@
 # <%= t('authentication.sign_in.email.heading') %>
```

Comment:
not sure if we need this comments

Reply:
This is an .erb file, so this is not a comment, it's a heading:

![Screenshot 2019-10-04 at 10 58 14](https://user-images.githubusercontent.com/1650875/66199137-e2bc0800-e695-11e9-9d31-2bd3ffa4e5d3.png)